src/core/etl/tests/Flow/ETL/Tests/Unit/Extractor/ExtractorTestCase.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Tests\Unit\Extractor;
+
+use Flow\ETL\{Config, Extractor, FlowContext, Row};
+use PHPUnit\Framework\TestCase;
+use function iterator_to_array;
+
+abstract class ExtractorTestCase extends TestCase
```

Comment:
If we decide that these methods do indeed help maybe it would a better idea to convert this function into a trait or something similar. The reason is that there are cases outside the Extractors tests where we counting or converting the rows.